api/models.py

```diff
@@ -43,7 +43,7 @@ def validate(cls, value):
         return ObjectId(value)
 
 
-class StateValues(enum.Enum):
+class StateValues(str, enum.Enum):
```

Comment:
How about directly using `StrEnum`?
https://docs.python.org/3/library/enum.html#enum.StrEnum

Reply:
> New in version 3.11

We're still on 3.10, but yes I guess that's something we could do when we upgrade.

Reply:
OK